tests/s3/test_object_tagging.py

```diff
@@ -1554,10 +1488,12 @@ def test_mpupload_valid_tagkeyspecialchars_42775(self):
         self.log.info("Retrieving tag of an object %s", self.object_name)
         resp = self.tag_obj.get_object_tags(self.bucket_name, self.object_name)
         assert_utils.assert_equal(
-            resp[1], {S3_OBJ_TST["test_9429"]["key"]: S3_OBJ_TST["test_9429"]["value"]}, resp[1])
+            resp[1][0], {'Key': f'{S3_OBJ_TST["test_9429"]["key"]}',
+                         'Value': f'{S3_OBJ_TST["test_9429"]["value"]}'}, resp[1])
         self.log.info("Retrieved tag of an object")
         self.log.info("Verify Multipart Upload with tag key having valid special characters")
 
+    @pytest.mark.parallel
```

Comment:
Let's not introduce this marker, unless you have verified parallel execution of the tests